packages/core/bootstrap/src/lib/cache/index.ts

```diff
@@ -114,13 +116,26 @@ export const withCache: Middleware = async (execute, options = defaultOptions())
   const _getRateLimitMaxAge = (data: AdapterRequest): number | undefined => {
     if (!data || !data.data) return
     if (isNaN(data.rateLimitMaxAge as number)) return
+    const feedId = data?.debug?.feedId
     const maxAge = Number(data.rateLimitMaxAge)
     if (maxAge && maxAge > ERROR_MAX_AGE) {
-      logger.warn(`Cache: Key TTL ERROR_MAX_AGE: Max Age is getting max values: ${maxAge} ms`, data)
+      logger.debug(
```

Comment:
Should these not be warn?

Reply:
It seems like there are scenarios where these will get thrown on every request just because of imbalanced feeds. It freaks nops out to see them.

I'll change them back though for now, let's see if other changes we make reduce it.